### ISSUE TITLE
Incorrect ld hl, $XXYY implementation ?

### ISSUE THREAD

**Issue by ISSOtm** [ISSOtm/gbz80-to-items#8]
```
ld hl, $D17C
ld a, $E0
ld (hl), a
sub a, $D9
dec hl
ld c, a
inc b
ld (hl), c
ret
```
This code doesn't compile correctly.
```
ThunderStone x124
TM09 x62
TM24 x119
TM14 x217
Secret Key x79
Poké Ball x113
TM01 x[any qty]
```
is expected, but
```
ThunderStone x62
TM24 x119
TM14 x217
Secret Key x79
Poké Ball x113
TM01 x[any qty]
```
is generated instead.

Apparently the argument to `ld hl, $D17C` is skipped entirely.
`ld`'s implementation is a bit of a crying shame, it must be a problem of instruction length - the instruction byte is correct (ThunderStone is output) but the arguments aren't present (2-byte offset)

May not affect `ld de, $XXYY` and `ld bc, $XXYY`, but if either suffers from the bug the other should, too.

**Comment by ISSOtm** [comment]
(Forgot to mention, but this was submitted by the folks at PRAMA Initiative)

**Comment by ISSOtm** [comment]
Fixed internally, it was a mistake of forgetting to read the arguments.

Commit coming soon.

**Comment by ISSOtm** [comment]
Fixed by commit 8587638.
